### ISSUE TITLE
`css.preprocessorMaxWorkers = true`

### ISSUE THREAD

**Issue by sapphi-red** [hi-ogawa/test-vite-sass#1]
Making an issue here instead as I feel it's a bit off-topic from the PR (https://github.com/vitejs/vite/pull/17728).

I saw this comment here.
https://github.com/hi-ogawa/test-vite-sass/blob/29411c86b39270c7ee2691d443dccb219630a173/vuetify/vite.config.ts#L9-L12
Just in case, did you put `css.preprocessorMaxWorkers = true`? Otherwise, adding that won't change anything.

Example: https://github.com/vitejs/vite/blob/66bdb1d7b41e46b5361606ff3811bdad6f625bcc/playground/css/vite.config.js#L85

**Comment by hi-ogawa** [comment]
Good point. I think I've toggled it a few times, but probably not in this example.
I updated README.md to include the build time with and without `preprocessorMaxWorkers`. The changes looks like this:

- `sass-embedded x modern` 575ms -> 937ms (slower)
- `sass x modern` 2.75s -> 1.76s (faster)
- `sass x legacy` 3.42s -> 2.04s (faster)

`sass` improved as expected, but `sass-embedded` might be suffering overhead of each dart process.
